### DISCUSSION TITLE
Great app

### DISCUSSION THREAD

**Discussion by AndiCee** [WilliamSchack/Spotify-Downloader#15] [General]
Hi @WilliamSchack 
I just signed up to GitHub to say thanks for making this very easy to use app. I was using a different one that downloaded directly from Spotify and they locked my account until I spoke with someone from Spotify, so I like how it downloads good quality songs elsewhere and I like the Normalize volume option and its pretty fast. Anyway... thanks again. :)

**Comment by WilliamSchack** [comment]
Thanks for using my program I really appreciate the support!

That is crazy that your account got locked from another app i am surprised that it's still even up if its getting peoples accounts closed, lucky I didn't go with that route... The songs in this program are actually downloaded from YouTube, I have put a lot of work into making sure that the songs are accurate and high quality and it has clearly paid off! It is still in active development as well so the features are only expanding from here with a few plans in the works for more upgrades and better usability so its only up from here!

Thanks again for using the program and if you have any questions or issues feel free to shoot me a message :)

**Reply by AndiCee** [reply]
I've tried quite a few over the years I'm pretty sure the one that was picked up by Spotify was called "AudKit Spotify Music Converter". They said that they had blocked my account because they detected suspicious behavior (possibly downloading) on my account. I denied it all of course. They made me change my password and unblocked the account. I've probably downloaded about 1000 songs using your app, so far all of them I have checked have been high quality and have been accurate. The music is downloaded from YouTube music? Interesting...  I download a lot of remixes, not very common stuff and only one song hasn't been available, so that's pretty impressive.

So you are doing an awesome job, thanks again. :)

**Comment by mailmefren** [comment]
Very good app. However, i got "Song not found" error at end of downloading process.

**Reply by WilliamSchack** [reply]
That happens when a song is not found, say when a song is on spotify but not on youtube it will not be able to download it because it cannot find a source on youtube to download it from. A good indicator to see if a song will be available to download will just be to search for it on youtube, but not all songs will be on their and the algorithm will pick up most but there are a few songs that slip through so there is really nothing I can do about this one sorry

Also if you have an issue in the future please create a new issue instead of commenting in the discussions :)

**Reply by mailmefren** [reply]
Thank you @WilliamSchack .

**Comment by xxTheR0xxx** [comment]
hi @WilliamSchack I'd like to say a very BIG THANK YOU here from Brazil !!! Your software is the BEST I've ever seen. There's no other available at the moment so functional as yours. Thank you so much !

**Reply by WilliamSchack** [reply]
Thanks for the support I am glad it helps!

**Comment by ian4kasablancas** [comment]
Hello @WilliamSchack, this program is incredible! It's helping me save time and is easy to use, just like @xxTheR0xxx said. Another BIG THANKS from Brazil! Since the use of phones was prohibited, many teens here in Brazil are downloading MP3 songs in multiple ways, such as transferring them to an SD card and putting them into an MP3 player so they can listen to music at school.

So, again, tysm! you are awesome!

**Comment by RensStoter** [comment]
man i trust this download more than those pesky little trash downloader websites. i really like using it cuz spotify is not on windowsxp so i gotta download them for my xp laptop

**Comment by trunkam** [comment]
hey man that's a very well done app , easy to use ! thanks a lot

**Comment by Santo131** [comment]
Привет @WilliamSchack я из России , очень долго искал как можно скачать свой плейлист за один раз , без танцев с бубном , твой продукт выше всяких похвал , я правда искал и пробовал разные варианты очень долго , на данный момент твое творение самое лучшее что я видел , наткнулся чисто случайно , у нас же все блокируют , не дают искать нужную и правдивую информацию , мы как в Корее )) спасибо тебе огромное , я бы хотел поддержать тебя материально , но к сожалению. не могу этого сделать , так как всевозможные переводы у нас под запретом. Но с удовольствие это сделаю когда ты покажешь нам , людям из России как это можно сделать . Огромное тебе спасибо за твой труд , СПАСИБО.

**Reply by WilliamSchack** [reply]
Thanks for the love!

Ive actually noticed that there are a fair few people from around the world using the app which is crazy to me but honestly makes me think that I should add translations to the app but since I dont know any languages other than english I might open it to the community to translate in the future which would definitely be helpful with v2 as the app will be much more complex

If you do want to support development you can on my Ko-Fi :) (https://ko-fi.com/williamschack)

Thanks again and have a good one!

---

Спасибо за поддержку!

Я заметил, что приложением пользуется довольно много людей со всего мира, что меня очень удивляет, но, честно говоря, это наводит на мысль о добавлении перевода. Однако, поскольку я не знаю других языков, кроме английского, я, возможно, в будущем открою возможность перевода для сообщества, что, безусловно, будет полезно для версии 2, так как приложение станет намного сложнее.

Если вы хотите поддержать разработку, вы можете сделать это на моём Ko-fi :) (https://ko-fi.com/williamschack)

Ещё раз спасибо и всего хорошего!
